apps/cowswap-frontend/src/modules/bridge/pure/contents/BridgingProgressContent/ReceivedBridgingContent/index.tsx

```diff
@@ -54,9 +58,20 @@ export function ReceivedBridgingContent({
         </b>
       </ConfirmDetailsItem>
 
-      <DepositTxLink depositTxHash={depositTxHash} sourceChainId={sourceChainId} />
-      {fillTxLink && (
-        <TransactionLinkItem link={fillTxLink} label="Destination transaction" chainId={destinationChainId} />
+      {explorerUrl ? (
+        <TransactionLinkItem
+          link={explorerUrl}
+          label="Bridge transaction"
+          chainId={0}
```

Comment:
Why chainId is 0?
Probably it's better to create a separate component for "Bridge transaction" instead of hacking TransactionLinkItem

Reply:
Refactored to a separate component now in https://github.com/cowprotocol/cowswap/pull/5976/commits/5611a4537c3e2e012c0a8ed5d94c42e78202f2e6